docs/cli-reference/ark_create_schedule.md

```diff
@@ -42,6 +42,7 @@ ark create schedule NAME --schedule="0 */6 * * *"
       --show-labels                                     show labels in the last column
       --snapshot-volumes optionalBool[=true]            take snapshots of PersistentVolumes as part of the backup
       --ttl duration                                    how long before the backup can be garbage collected (default 720h0m0s)
+  -w, --wait                                            wait for the operation to complete
```

Comment:
we don't want this flag on `ark schedule create`. It looks like it's getting inherited because the code assumes anything that's an option for creating a backup, should be an option for creating a schedule too (https://github.com/heptio/ark/blob/master/pkg/cmd/cli/schedule/create.go#L76 and https://github.com/heptio/ark/blob/master/pkg/cmd/cli/schedule/create.go#L81 do it).  Need to find a way to separate that out so it applies for backup creates but not schedule creates.